src/controllers/videoosd.js

```diff
@@ -1071,6 +1071,12 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
         }
 
         function onWindowKeyDown(e) {
+            // FIXME: Conflicts with keyboard navigation
+            // FIXME: Then the keyboard is completely ignored. Need another solution.
```

Comment:
this doesn't sound pretty... I would rather we have a proper navigation for all modes - AFAIK TV remote navigation is very similar to normal keyboard, but maybe it just uses another key codes - those should be extracted in a single file and defined there depending on context then

Reply:
Codes of navigation buttons the same as keyboard arrow keys ones. When I press a key on the remote control, it controls a playback, and I cannot navigate in video player toolbar. Actually, it navigate and rewind/forward in parallel. For playback control I use media buttons on the remote control. For now, I have no idea how it should be. So this needs to be discussed. Maybe playback control must be active when focused on video player itself with hidden toolbar. And we have to press some key to show up the osd.